README.md

```diff
@@ -0,0 +1,69 @@
+# hello-kubernetes
+Example end-to-end build, test, and deploy pipelines for Kubernetes, using Google Cloud Build.
+
+# Instructions
+
+If you are new to Google Cloud Build, we recommend you start by visiting the [manage resources page](https://console.cloud.google.com/cloud-resource-manager) in the Cloud Console, [learn how to enable billing](https://cloud.google.com/billing/docs/how-to/modify-project), [enable the Cloud Build API](https://console.cloud.google.com/flows/enableapi?apiid=cloudbuild.googleapis.com), and [install the Cloud SDK](https://cloud.google.com/sdk/docs/).
+
+## Continuous integration testing
+
+Google Cloud Build supports continuous integration testing, with a test running automatically every time a Pull Request is created or updated in GitHub.
+
+To try this out for yourself with a simple Go application which prints a greeting:
+
+* Fork this repo on GitHub by clicking the "Fork" icon in the top right-hand corner of the screen.
+* Open the [Google Cloud Build GitHub app](https://github.com/apps/google-cloud-build), click Install, and follow the setup instructions.  Select the `hello-kubernetes` repository you just created.
+* Back in your forked repo in GitHub, click to open `main_test.go`.  You will see some Go tests which make sure that the greeting matches the name provided.
+* Click the pencil icon in the top right-hand corner of the screen.
+* Edit one of the greetings so that the greeting does not match the name.
+* At the bottom of the screen, choose "Create a new branch and start a pull request".  Click "Propose file change".
+* On the page which opens, click "Create pull request".
+* You should see a message: "All checks have failed".  If you wish, you can see the build log by clicking the "Details" button and then selecting the link at the bottom of the screen which says "View more details on Google Cloud Build".
```

Comment:
Suggest "...selecting the link in small print at the bottom of the screen..." -- I had trouble finding it.

Reply:
Done